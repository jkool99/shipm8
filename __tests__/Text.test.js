import React from 'react';
import { View, Text } from 'react-native';
import renderer from 'react-test-renderer';

describe('Text', () => {
  it('renders correctly', () => {
    const instance = renderer
      .create(
        <View style={{}}>
          <Text>React-Native Text Test</Text>
        </View>,
      )
      .toJSON();

    expect(instance).toMatchSnapshot();
  });
});
