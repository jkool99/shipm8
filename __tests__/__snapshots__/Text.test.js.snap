// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders correctly 1`] = `
<View
  style={Object {}}
>
  <Text>
    React-Native Text Test
  </Text>
</View>
`;
